src/App.jsx

```diff
@@ -14,33 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const initialGoods = goodsFromServer;
+  const [goods, setGoods] = useState([...initialGoods]);
```

Comment:
- You don't need to store goods in a state
- `initialGoods` variable changes on every rerender